guvnor-project/guvnor-project-client/src/main/java/org/guvnor/common/services/project/client/POMEditorPanel.java

```diff
@@ -58,18 +59,21 @@ public void setPOM( POM model,
 
         view.setName( model.getName() );
         view.setDescription( model.getDescription() );
+        view.enableGroupID();
+        view.enableArtifactID();
+        view.enableVersion();
+
         if ( model.hasParent() ) {
             view.setParentGAV( model.getParent() );
             view.showParentGAV();
-            view.disableGroupID( "" );
-            view.enableArtifactID();
-            view.disableVersion( "" );
+            if ( !isChildGAVEditionEnabled() ) {
```

Comment:
Your use of the word "edition" is wrong (but then again, don't even ask me what it is is Portuguese!) Personally I'd replace all occurrences of "edition" with "edit" e.g. ```isChildGAVEditEnabled()``` etc